### PR TITLE
Create event emitters on adding and removing of transactions for transaction pool - Closes #1013

### LINKED ISSUE #1013: Create event emitters on adding and removing of transactions for transaction pool
### Expected behavior
Transaction pool should emit events with specific actions when transactions are added or removed from the transaction pool.
### Actual behavior
Transaction pool does not fire any events.

### PR DESCRIPTION
**Pull request by SargeKhan**
### What was the problem?
The transaction pool was not firing any events when transactions were added or removed from the transaction pool.
### How did I fix it?
Created two events which are fired every time transactions are added or removed from the transaction pool. These events are:
1- transactionAdded
2- transactionRemoved

### Review checklist

* The PR resolves #1013 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: packages/lisk-transaction-pool/src/transaction_pool.ts [hunk 1]
```diff
@@ -213,6 +218,12 @@ export class TransactionPool {
 		this._queues.validated.enqueueMany(removedTransactionsByRecipientId);
 		// Add transactions to the verified queue which were included in the verified removed transactions
 		this._queues.verified.enqueueMany(transactions);
+
+		this.emit(EVENT_ADDED_TRANASCTION, {
+			action: 'addVerifiedRemovedTransactions',
```
Comment (shuse2) [inline packages/lisk-transaction-pool/src/transaction_pool.ts]: maybe action should be constant and exported?

**Review by shuse2 — CHANGES_REQUESTED** [review]
looks good, just minor comment

File: packages/lisk-transaction-pool/src/transaction_pool.ts [hunk 2]
```diff
@@ -99,7 +100,22 @@ const DEFAULT_VALIDATED_TRANSACTIONS_LIMIT_PER_PROCESSING = 100;
 const DEFAULT_VERIFIED_TRANSACTIONS_PROCESSING_INTERVAL = 30000;
 const DEFAULT_VERIFIED_TRANSACTIONS_LIMIT_PER_PROCESSING = 100;
 
-export class TransactionPool {
+export const EVENT_ADDED_TRANASCTIONS = 'transactionsAdded';
```
Comment (mitsuaki-u) [inline packages/lisk-transaction-pool/src/transaction_pool.ts]: typo

File: packages/lisk-transaction-pool/src/transaction_pool.ts [hunk 3]
```diff
@@ -343,17 +371,30 @@ export class TransactionPool {
 
 		this._queues[queueName].enqueueOne(transaction);
 
+		this.emit(EVENT_ADDED_TRANASCTIONS, {
```
Comment (mitsujutsu) [inline packages/lisk-transaction-pool/src/transaction_pool.ts]: same as above

**Review by shuse2 — APPROVED** [review]

**Pull request merged**